solr/contrib/scripting/src/java/org/apache/solr/scripting/util/xslt/XSLTParams.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.solr.scripting.util.xslt;
+
+
+/**
+ * Parameters used across the XSLT related classes.
+ */
+public interface XSLTParams {
```

Comment:
I see why you added this here but I don't think there's a role for XSLTParams.  Just move this "tr" constant to the plugin that defines it for its internal use only.  CommonParams was in SolrJ deliberately; This XSLTParams you added is not (and should not be).